notebooks/254-llm-chatbot/254-llm-chatbot.ipynb

```diff
@@ -710,7 +815,13 @@
     "from optimum.intel.openvino import OVModelForCausalLM\n",
     "from transformers import AutoTokenizer\n",
     "\n",
-    "model_dir = Path(model_id.value) / (\"FP16\" if not int8_compressed_weights.value else \"INT8_compressed_weights\")\n",
+    "if prepare_int4_model.value:\n",
```

Comment:
On the previous stage it is possible to choose all compression model types. But here we get the first chosen. Looks like it is make sense to add widget for choosing what type of converted models we want to use.

Reply:
Added a dropdown for selecting a model